### DISCUSSION TITLE
Combat bracelet(6) not recognised for elite emote clue step

### DISCUSSION THREAD

**Discussion by DrYoshiyahu** [runelite/runelite#16514] [Ideas]
![image](https://user-images.githubusercontent.com/14756430/233269067-1759e90c-c86e-46af-852b-a532b4e20389.png)

Location: slayer tower. The item is Combat bracelet(6)

**Comment by DrYoshiyahu** [comment]
My apologies, I thought I was posting this in Issues, not Ideas! 😓

**Comment by Nightfirecat** [comment]
Created #16515 to track this.
